*** CMakeLists.txt	2013-10-03 15:15:13.830056708 +0200
--- CMakeLists.txt.old	2013-10-01 11:53:21.000000000 +0200
***************
*** 23,29 ****
  
  add_executable(isiviewer ${SOURCES})
  
! target_link_libraries (isiviewer GLEW ${QT_LIBRARIES} QGLViewer GL GLU)
  
  # add a symbolic link toward the source directory
  add_custom_command(TARGET isiviewer POST_BUILD
--- 23,29 ----
  
  add_executable(isiviewer ${SOURCES})
  
! target_link_libraries (isiviewer GLEW ${QT_LIBRARIES} QGLViewer) 
  
  # add a symbolic link toward the source directory
  add_custom_command(TARGET isiviewer POST_BUILD
